src/components/core/theme-context/theme-provider.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, useMemo, useState } from 'react';
+import ThemeContext, { EColorMode, ThemeContextValueType } from './theme-context';
+
+const ThemeProvider: FC = ({ children }) => {
```

Comment:
React.FC is going to be deprecated - https://spin.atomicobject.com/2022/01/04/think-twice-react-fc/
Can we declare custom type instead of FC?

Reply:
done. thanks :pray: